crates/sui-proc-macros/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub fn init_static_initializers(_args: TokenStream, item: TokenStream) -> TokenS
 
                     register_package_hooks(Box::new(SuiPackageHooks {}));
                     let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-                    path.extend(["..", "..", "examples", "move", "basics"]);
+                    path.push(env!("SIMTEST_STATIC_INIT_MOVE"));
```

Comment:
It feels a bit unintuitive to have the `SIMTEST_STATIC_INIT_MOVE` as a relative path although it's probably useful because it allows storing the files somewhere in the repo. 
Can we make this clear from the variable name? For example, `SIMTEST_STATIC_INIT_MOVE_RELATIVE_PATH`, although that's really a mouthful... 🤷 

Also, we could do something like `env::var("...").unwrap_or("../../examples/move/basics")`, then we don't have to set this explicitly in this repo.

Reply:
Thanks @mlegner and they all make sense! Seems taking Mark's suggestion to use absolute path is more intuitive. I didn't find a way to get the project root dir in rust, so right now I still set it in the script. It's also easy to test so that when we set it in walrus, we know it should work.